src/exojax/spec/opacont.py

```diff
@@ -27,11 +27,14 @@ class OpaCIA(OpaCont):
     """Opacity Continuum Calculator Class for CIA
 
     """
-    def __init__(self, cdb, nu_grid):
+    def __init__(self, cdb, nu_grid, wavelength_order="descending"):
         self.method = "cia"
         self.warning = True
         self.nu_grid = nu_grid
-        self.wav = nu2wav(self.nu_grid, unit="AA")
+        self.wavelength_order = wavelength_order
+        self.wav = nu2wav(self.nu_grid,
+                          wavelenght_order=self.wavelength_order,
```

Comment:
I found that there was a typo "waveleng'ht'_order".

Reply:
Thanks!